notebooks/algorithms/phase-kickback.ipynb

```diff
@@ -0,0 +1,566 @@
+       "Q 3647 3509 3647 2328 \n",
+       "Q 3647 1150 3233 529 \n",
+       "Q 2819 -91 2034 -91 \n",
+       "Q 1250 -91 836 529 \n",
+       "Q 422 1150 422 2328 \n",
+       "Q 422 3509 836 4129 \n",
+       "Q 1250 4750 2034 4750 \n",
+       "z\n",
+       "\" transform=\"scale(0.015625)\"/>\n",
+       "      </defs>\n",
+       "      <use xlink:href=\"#DejaVuSans-Oblique-71\"/>\n",
+       "      <use xlink:href=\"#DejaVuSans-30\" transform=\"translate(63.476562 -16.40625)scale(0.7)\"/>\n",
+       "     </g>\n",
+       "    </g>\n",
+       "   </g>\n",
+       "   <g id=\"text_2\">\n",
+       "    <g clip-path=\"url(#pb03c8b0eb8)\">\n",
+       "     <!-- ${q}_{1}$ -->\n",
+       "     <g transform=\"translate(36.253371 93.236799)scale(0.1625 -0.1625)\">\n",
+       "      <defs>\n",
+       "       <path id=\"DejaVuSans-31\" d=\"M 794 531 \n",
+       "L 1825 531 \n",
+       "L 1825 4091 \n",
+       "L 703 3866 \n",
+       "L 703 4441 \n",
+       "L 1819 4666 \n",
+       "L 2450 4666 \n",
+       "L 2450 531 \n",
+       "L 3481 531 \n",
+       "L 3481 0 \n",
+       "L 794 0 \n",
+       "L 794 531 \n",
+       "z\n",
+       "\" transform=\"scale(0.015625)\"/>\n",
+       "      </defs>\n",
+       "      <use xlink:href=\"#DejaVuSans-Oblique-71\"/>\n",
+       "      <use xlink:href=\"#DejaVuSans-31\" transform=\"translate(63.476562 -16.40625)scale(0.7)\"/>\n",
+       "     </g>\n",
+       "    </g>\n",
+       "   </g>\n",
+       "  </g>\n",
+       " </g>\n",
+       " <defs>\n",
+       "  <clipPath id=\"pb03c8b0eb8\">\n",
+       "   <rect x=\"7.2\" y=\"7.2\" width=\"110.722271\" height=\"109.082399\"/>\n",
+       "  </clipPath>\n",
+       " </defs>\n",
+       "</svg>\n"
+      ],
+      "text/plain": [
+       "<Figure size 146.652x144.48 with 1 Axes>"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit import QuantumCircuit\n",
+    "qc = QuantumCircuit(2)\n",
+    "qc.cz(0,1)\n",
+    "qc.draw()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The CNOT Gate\n",
+    "\n",
+    "Let's look at the phase kickback effect with a different two-qubit gate. Since the $|{-}\\rangle$ state is an eigenstate of the X-gate, with eigenvalue $-1$, we get:\n",
+    "\n",
+    "$$ \\class{_matrix-CX}{CX}\\class{_eig-vec-X}{|{-}\\rangle}\\class{_control-qubit-pre}{(\\alpha|0\\rangle + \\beta|1\\rangle)} = \\class{_eig-vec-X}{|{-}\\rangle}\\class{_control-qubit-post}{(\\alpha|0\\rangle - \\beta |1\\rangle)} $$\n",
+    "\n",
+    "![](images/kickback/cnot-kickback.svg)\n",
+    "\n",
+    "Again, in this case the phase change $\\theta = 0.5$, so our control qubit is flipped around the Z-axis.\n",
+    "\n",
+    "<!-- ::: q-block -->\n",
+    "\n",
+    "### Worked example\n",
+    "\n",
+    "<details>\n",
+    "    <summary>Kickback with the CNOT gate(click to expand)</summary>\n",
```

Comment:
Tiny missing space here `CNOT gate(click to expand)`

should be: `CNOT gate (click to expand)`